.github/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
 
 <h3>Improvements</h3>
 
+* Add QubitMatrix operator for custom operator implementation
```

Comment:
```suggestion
* Add `QubitMatrix` operation for custom operator implementation
```

Reply:
I know I originally suggested `QubitMatrix`, but I realize now it doesn't really explain _what_ it is. Are there any better suggestions? All I can think of is `QubitOperator` (a la `QubitUnitary`, but without the implied unitarity)

Reply:
`QubitOperator` makes most sense to me, I've made the change here: c1a9cb2969654729bda9f53679f9978334184371